client/battle/BattleActionsController.cpp

```diff
@@ -914,6 +910,8 @@ bool BattleActionsController::isCastingPossibleHere(const CSpell * currentSpell,
 	const spells::Mode mode = heroSpellToCast ? spells::Mode::HERO : spells::Mode::CREATURE_ACTIVE;
 
 	spells::Target target;
+	if(sourceIsFisrtTarget)
+		target.emplace_back(targetStack);

```

Comment:
Instead of bool parameters perhaps simply check it for null?
```suggestion
	if(targetStack)
		target.emplace_back(targetStack);
```

Reply:
done